contracts/nft/mainnet/NFTMainnetBridgingToPolygonFacet.sol

```diff
@@ -21,10 +21,13 @@ import {
 
 // Storage
 import { NFTStorageLib, NFTStorage } from "../../storage/nft.sol";
+import "hardhat/console.sol";
 
 contract NFTMainnetBridgingToPolygonFacet {
     address private constant POLYGON_NFT_ADDRESS =
         0x83AF2b36A3F8593203b2098CBec616A57f1A80cC;
+    address private constant ERC721_PREDICATE =
```

Comment:
We do not need the **ERC721** predicate because we are only depositing V2 (**ERC1155**)